dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/ConnectorArgumentsTest.java

```diff
@@ -61,11 +62,33 @@ public void getDatabases_trimDatabaseNames() throws IOException {
   @Test
   public void getDatabases_trimDatabaseNamesFilteringOutBlankStrings() throws IOException {
     ConnectorArguments arguments =
-        new ConnectorArguments(
-            new String[] {"--connector", "teradata", "--database", "db1, ,,, db2 "});
+        new ConnectorArguments("--connector", "teradata", "--database", "db1, ,,, db2 ");
 
     List<String> databaseNames = arguments.getDatabases();
 
     assertEquals(ImmutableList.of("db1", "db2"), databaseNames);
   }
+
+  @Test
+  public void testNoUserFlag() throws IOException {
+    ConnectorArguments arguments = new ConnectorArguments("--connector", "abcABC123");
+    Runnable action = () -> arguments.getUserOrFail();
+    String expectedMessage =
```

Comment:
The variable is used only once in the Assert step. It is not needed in the Arrange step. I would suggest inlining.